package/gluon-mesh-batman-adv/files/etc/config/batman-adv

```diff
@@ -0,0 +1,20 @@
+
+config 'mesh' 'bat0'
+	option 'aggregated_ogms'
+	option 'ap_isolation'
+	option 'bonding'
+	option 'fragmentation'
+	option 'gw_bandwidth'
+	option 'gw_mode'
+	option 'gw_sel_class'
+	option 'log_level'
+	option 'orig_interval'
+	option 'bridge_loop_avoidance'
+	option 'distributed_arp_table'
+	option 'multicast_mode'
+	option 'network_coding'
+	option 'hop_penalty'
+	option 'isolation_mark'
+
+# yet another batX instance
+# config 'mesh' 'bat5'
```

Comment:
Moving this file to gluon-mesh-batman-adv is quite ugly. If we really want to merge coexistence before updating to batadv v2019.2 (and thus before Gluon v2019.1), splitting this into `/etc/config/batman-dav` and `/etc/config/batman-adv-legacy` would seem preferable to me - but we could also defer this until after the release.